gateway/src/apicast/policy/jwt_claim_check/jwt_claim_check.lua

```diff
@@ -0,0 +1,55 @@
+local policy = require('apicast.policy')
+local _M = policy.new('JWT check policy')
+
+local Operation = require('apicast.conditions.operation')
+-- local Condition = require('apicast.conditions.condition')
+
+local new = _M.new
+
+local default_error_message = "Request block due JWT claim policy"
+
+local function deny_request(error_msg)
+  ngx.status = ngx.HTTP_FORBIDDEN
+  ngx.say(error_msg)
+  ngx.exit(ngx.status)
+end
+
+
+local function verdict(check_type, status)
+  -- Default whitelist mode, if not valid using whitelist
+  if check_type == "blacklist" then
+    return not status
+  end
+
+  return status
+end
+
+function _M.new(config)
+  local self = new(config)
+  self.error_message = config.error_message or default_error_message
+
+  self.conditions = {}
+  for _,condition in ipairs(config.operations) do
```

Comment:
Can you declare `ipairs` as a local at the beginning of the file? Same with `pairs` used below.